src/kili/adapters/kili_api_gateway/mixin.py

```diff
@@ -0,0 +1,38 @@
+"""Kili API Gateway module for interacting with Kili."""
```

Comment:
it is not a mixin. It can be named `kili_api_gateway.py`, then `KiliApiGateway` can be imported in the `__init__.py`.

Reply:
better not put the class KiliApiGateway in the `__init__`, else, everytime we want to get a specific class or method from the gateway, it will go through the `__init__`, which will import all sub-packages, which causes circular imports issues